src/components/PluginReleases.jsx

```diff
@@ -2,52 +2,65 @@ import React, {useState, useEffect} from 'react';
+
 import {formatter} from '../commons/helper';
 
+import './PluginReleases.css';
 
-function PluginReleases({pluginId}) {
-    const [isLoading, setIsLoading] = useState(false);
+
+function PluginReleases({pluginId, versions}) {
     const [releases, setReleases] = useState([]);
 
     useEffect(() => {
         const fetchData = async () => {
-            setIsLoading(true);
-            const result = await axios(`/api/plugin/${pluginId}/releases`);
-            setReleases(result.data.releases || []);
-            setIsLoading(false);
+            try {
+                const result = await axios(`${process.env.GATSBY_API_URL || '/api'}/plugin/${pluginId}/releases`);
+                let releases = [];
+                if (result && result.data && result.data.releases) {
+                    releases = result.data.releases;
+                }
+                setReleases(releases);
+            } catch (e) {
+                console.error('fetch api release data', e); // eslint-disable-line no-console
+            }
         };
         fetchData();
         return;
     }, []);
-
-    if (isLoading) {
-        return (<div className="spinner-wrapper">
-            <div className="spinner-border" role="status">
-                <span className="sr-only">Loading...</span>
-            </div>
-        </div>);
+    
+    if (!versions) {
+        return (<div id="pluginReleases--container" className="container" />);
     }
-
+    
     return (
         <div id="pluginReleases--container" className="container">
-            {releases && releases.map(release => {
+            {versions.map(version => {
+                const release = releases.find(release => version.version === release.tag_name.replace(/^[^0-9]*/, '')) || {};
                 return (
-                    <div key={release.tag_name} className="item card">
+                    <div key={version.id} className="item card">
                         <div className="card-header">
                             <h5 className="card-title d-flex justify-content-between">
-                                <div>{release.name || release.tag_name}</div>
+                                <div><a href={version.url}>{release.name || version.version}</a></div>
                                 <div>
-                                    {'Released: '}
-                                    <TimeAgo date={new Date(release.published_at)} formatter={formatter}/>
+                                    <TimeAgo date={new Date(version.buildDate)} formatter={formatter}/>
                                 </div>
                             </h5>
                         </div>
                         <div className="card-body">
-                            <p
+                            <div>
+                                <InstallViaCLI pluginId={pluginId} version={version.version} />
+                            </div>
+                            {release.bodyHTML && (<blockquote
```

Comment:
I'm not a huge fan of this block quote bit, I think we should remove it,

especially I don't like the italics
https://plugin-site-regud.ondigitalocean.app/slack/#releases

![image](https://user-images.githubusercontent.com/21194782/100713618-79747980-33ac-11eb-9ac3-2b177c112cca.png)

Reply:
> I'm not a huge fan of this block quote bit, I think we should remove it,

and replace it with? just the <p> directly doesn't really work, it blends everything together, though open to ideas with spacing and stuff

Reply:
what if we just move the install via cli to after the notes and before the checksums?